README

```diff
@@ -1,5 +1,9 @@
-GREG Please move this info to the associated examples.
+GREG:
 
+1) Please move this info into the associated examples.
+2) Look at rdbService and serviceAP
+I left these almost untouched from release 4.5
+3) Look at KNOWNPROBLEMS.md
```

Comment:
On 03/25/2016 08:41 AM, Ralph Lange wrote:
>
> In README 
> <https://github.com/epics-base/exampleJava/pull/3#discussion_r57440755>:
>
> >
> > +1) Please move this info into the associated examples.
> > +2) Look at rdbService and serviceAP
> > +I left these almost untouched from release 4.5
> > +3) Look at KNOWNPROBLEMS.md
>
> /Same as above:/
> Please open an issue for this.
> I don't think a remark in a known problems file is enough to trigger 
> an action of someone else.
>

The known problems covers a range of issues.
Since the examples involve many modules from both Java and C++.
Thus for several of the problems it is not obvious which modules is 
causing the problem so not clear which modules should have an issue 
assigned to it.

My next task is make minor changes to exampleCPP resulting from working 
on exampleJava.

Then I will do some tests using both a Java server and a C++ server.
The tests will consist of both Java and C++ clients each communicating 
with both a Java Server and a C++ server.

Then I may have a better idea of which module is causing each problem.

With regards to asking Greg to decide what to do with the original 
examples that were in exampleJava,
I don't think it makes sense to open an issue until after the merge.

By the way it looks like Matej has already fixed the BaseV3Channel problem.




> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub 
> <https://github.com/epics-base/exampleJava/pull/3/files/da20bc1570d91dbe75b2c1b7cac4eafe91fe0f46..dfadef70018eb8efac858c3b6695fa82464430e9#r57440755>
>